modules/Bio/EnsEMBL/Compara/RunnableDB/PairAligner/ParsePairAlignerConf.pm

```diff
@@ -901,10 +926,12 @@ sub write_mlss_entry {
     my $ref_name;
     my $name;
 
-    foreach my $species_name ($ref_genome_db->name, $non_ref_genome_db->name) {
+    foreach my $gdb ($ref_genome_db, $non_ref_genome_db) {
+        my $species_name = $gdb->name;
         $species_name =~ s/\b(\w)/\U$1/g;
-        $species_name =~ s/(\S)\S+\_/$1\./;
+        $species_name =~ s/(\S)\S+\_/$1/;
```

Comment:
I wanted to change that method to use Utils::MasterDatabase instead but didn't have a way of testing it. Can you share the `init_pipeline` command you used ?

Reply:
```
init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::Sprouts::Lastz_conf -mlss_id_list "[1,2,3,4]" -pipeline_name sprouts_lastz -host mysql-ens-compara-prod-8 -port 4618
```
The _sprouts_ division and all its config files are at `/homes/jalvarez/workbench/src/dev/ensembl-compara`.